db/re/mob_db.conf

```diff
@@ -81931,15 +81931,353 @@ mob_db: (
+	}
+	ViewRange: 10
+	ChaseRange: 12
+	Size: "Size_Small"
+	Race: "RC_Plant"
+	Element: ("Ele_Wind", 1)
+	Mode: {
+		CanMove: true
+		Aggressive: true
+		CastSensorIdle: true
+		CanAttack: true
+		ChangeTargetMelee: true
+		ChangeTargetChase: true
+	}
+	MoveSpeed: 300
+	AttackDelay: 1500
+	AttackMotion: 500
+	DamageMotion: 1000
+	Drops: {
+		Mould_Powder: 3000
+		Yellow_Gemstone: 500
+		Starsand_Of_Witch: 500
+		Moth_Dust: 1500
+		Hood_: 8
+		Baby_Pacifier: 50
+		Neo_Punk_Card: 1
+	}
+},
+{
+	Id: 2920
+	SpriteName: "ARC_ELDER"
+	Name: "Arc Elder"
+	Lv: 168
+	Hp: 293640
+	Sp: 1
+	Exp: 9086
+	JExp: 17532
+	AttackRange: 3
+	Attack: [2513, 802]
+	Def: 105
+	Mdef: 41
+	Stats: {
+		Str: 100
+		Agi: 63
+		Vit: 35
+		Int: 99
+		Dex: 106
+		Luk: 61
+	}
+	ViewRange: 10
+	ChaseRange: 12
+	Size: "Size_Large"
+	Race: "RC_DemiHuman"
+	Element: ("Ele_Neutral", 4)
+	Mode: {
+		CanMove: true
+		Aggressive: true
+		CanAttack: true
+		Angry: true
+		ChangeTargetMelee: true
+		ChangeTargetChase: true
+	}
+	MoveSpeed: 165
+	AttackDelay: 1552
+	AttackMotion: 1152
+	DamageMotion: 336
+	Drops: {
+		Old_Magic_Circle: 3000
+		Rent_Spell_Book: 1000
+		Encyclopedia: 5
+		Wizardy_Staff: 1
+		Old_Card_Album: 2
+		Twisted_Key_of_Time: 1500
+		Elder_Branch: 500
+		Arc_Elder_Card: 1
+	}
+},
+{
+	Id: 2921
+	SpriteName: "OWL_VISCOUNT"
+	Name: "Owl Viscount"
+	Lv: 168
+	Hp: 295240
+	Sp: 1
+	Exp: 9086
+	JExp: 11089
+	AttackRange: 1
+	Attack: [4368, 900]
+	Def: 113
+	Mdef: 45
+	Stats: {
+		Str: 87
+		Agi: 51
+		Vit: 45
+		Int: 88
+		Dex: 106
+		Luk: 50
+	}
+	ViewRange: 10
+	ChaseRange: 12
+	Size: "Size_Large"
+	Race: "RC_Demon"
+	Element: ("Ele_Neutral", 3)
+	Mode: {
+		CanMove: true
+		Aggressive: true
+		CastSensorIdle: true
+		Boss: true
+		CanAttack: true
+		Detector: true
+		CastSensorChase: true
+		ChangeChase: true
+		ChangeTargetMelee: true
+		ChangeTargetChase: true
+	}
+	MoveSpeed: 195
+	AttackDelay: 1345
+	AttackMotion: 824
+	DamageMotion: 440
+	Drops: {
+		Tatters_Clothes: 2500
+		Soft_Feather: 1000
+		Monocle_: 1
+		Crystal_Mirror: 1
+		Pocket_Watch__: 2
+		Quadrille_: 1
+		Menswear: 1
+		Owl_Viscount_Card: 1
+	}
+},
 //2922,G_OWL_VISCOUNT
-//2923,OWL_MARQUEES
-
+{
+	Id: 2923
+	SpriteName: "OWL_MARQUEES"
+	Name: "Owl Marquis"
+	Lv: 170
+	Hp: 630000
+	Sp: 1
+	Exp: 21747
+	JExp: 13806
+	AttackRange: 2
+	Attack: [1887, 603]
+	Def: 127
+	Mdef: 25
+	Stats: {
+		Str: 112
+		Agi: 65
+		Vit: 55
+		Int: 102
+		Dex: 108
+		Luk: 72
+	}
+	ViewRange: 10
+	ChaseRange: 12
+	Size: "Size_Large"
+	Race: "RC_Demon"
+	Element: ("Ele_Neutral", 3)
+	Mode: {
+		CanMove: true
+		Aggressive: true
+		CastSensorIdle: true
+		Boss: true
+		CanAttack: true
+		Detector: true
+		CastSensorChase: true
+		ChangeChase: true
+		ChangeTargetMelee: true
+		ChangeTargetChase: true
+	}
+	MoveSpeed: 175
+	AttackDelay: 1345
+	AttackMotion: 824
+	DamageMotion: 440
+	Drops: {
+		Tatters_Clothes: 2000
+		Soft_Feather: 1500
+		Kakkung_: 1
+		Staff_Of_Soul: 1
+		Menswear: 3
+		Sword_Stick: 1
+		Monocle_: 1
```

Comment:
according to kRO website it drops the item 18874 which in the item db Cyclops_Glasses !!